src/Compilers/CSharp/Test/Semantic/Semantics/InlineArrayTests.cs

```diff
@@ -8591,12 +8703,9 @@ public struct Buffer10
 
             var compilation = CreateCompilation(source, targetFramework: TargetFramework.NetCoreApp);
             compilation.VerifyDiagnostics(
-                // (8,2): error CS0181: Attribute constructor parameter 'x' has type 'Span<int>', which is not a valid attribute parameter type
-                // [A(default(Buffer10))]
-                Diagnostic(ErrorCode.ERR_BadAttributeParamType, "A").WithArguments("x", "System.Span<int>").WithLocation(8, 2),
-                // (8,4): error CS9501: Cannot convert expression to 'Span<int>' because it is not an assignable variable
+                // (8,4): error CS1503: Argument 1: cannot convert from 'Buffer10' to 'System.Span<int>'
                 // [A(default(Buffer10))]
-                Diagnostic(ErrorCode.ERR_InlineArrayConversionToSpanNotSupported, "default(Buffer10)").WithArguments("System.Span<int>").WithLocation(8, 4)
+                Diagnostic(ErrorCode.ERR_BadArgType, "default(Buffer10)").WithArguments("1", "Buffer10", "System.Span<int>").WithLocation(8, 4)
```

Comment:
Now I wonder what happens if there is a user-defined conversion from `Buffer10` to `System.Span<int>`?

Reply:
> Now I wonder what happens if there is a user-defined conversion from `Buffer10` to `System.Span<int>`?

User defined conversions are not allowed in attributes or default values.